java-security/src/main/java/com/sap/cloud/security/config/Environments.java

```diff
@@ -49,4 +74,8 @@ public static Environment readFromInput(InputStream input) {
 		return CFEnvironment.getInstance(str -> vcapServices.toString(), str -> null);
 	}
 
+	private static boolean isK8sEnv() {
```

Comment:
is it required to check that again, otherwise the result should be remembered in a static variable